examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -66,55 +67,64 @@ export class TaskList extends DataObject implements ITaskList {
      * collection pattern -- see the contact-collection example for more details on this pattern.
      */
     private readonly tasks = new Map<string, Task>();
+    /*
+    * savedData stores data retrieved from the external source.
+    */
+    private _savedData: SharedMap | undefined;
+    /*
+    * draftData is used for storage of the draft Fluid data. It's used with savedData
+    * to resolve & synchronize the data.
+    * TODO: Update^ when the sync mechanism is appropriately defined.
+    */
+    private _draftData: SharedMap | undefined;
+
+    private get savedData(): SharedMap {
+        if (this._savedData === undefined) {
+            throw new Error("The savedData SharedMap has not yet been initialized.");
+        }
+        return this._savedData;
+    }
+
+    private get draftData(): SharedMap {
+        if (this._draftData === undefined) {
+            throw new Error("The draftData SharedMap has not yet been initialized.");
+        }
+        return this._draftData;
+    }
 
     public readonly addTask = (id: string, name: string, priority: number): void => {
+
         if (this.tasks.get(id) !== undefined) {
             throw new Error("Task already exists");
         }
-        const nameString = SharedString.create(this.runtime);
-        nameString.insertText(0, name);
-        const priorityCell = SharedCell.create(this.runtime) as ISharedCell<number>;
-        priorityCell.set(priority);
+        const savedNameString = SharedString.create(this.runtime);
+        const draftNameString = SharedString.create(this.runtime);
+
+        savedNameString.insertText(0, name);
```

Comment:
This is good for now -- one thing we'll eventually need to think about though is that assuming `addTask` is called for tasks being added in Fluid, we should only be writing to the draft map directly here (and then the saved map will get updated when the data syncs back).  I think it will be easiest to make that change once we have Kajari's portion available.  Maybe add a TODO here to note that.